components/RootLayout.vue

```diff
@@ -56,7 +56,7 @@ export default class RootLayout extends Vue {
 
   async mounted() {
     this.initUIStore()
-    await this.restoreSession()
+    await this.restoreSessionIfNecessary()
```

Comment:
is it better to let `this.isOpenChainUpgradeBlockingDialog = !!IS_CHAIN_UPGRADING` execute first?